plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -6,57 +6,59 @@ import Breadcrumb from "#SRC/js/components/Breadcrumb";
 import BreadcrumbTextContent from "#SRC/js/components/BreadcrumbTextContent";
 import PageHeaderBreadcrumbs from "#SRC/js/components/PageHeaderBreadcrumbs";
 
-export default function Breadcrumbs({ item, children, states }) {
-  function getBreadcrumb(item, id = "", name = "") {
-    const isDetailPage = [...item.path, item.name].join(".") === id;
-    const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
-
+export default function Breadcrumbs({ path, name, children, extra }) {
+  /**
+   * @param  {string} link
+   * @param  {string[]} pathname path including name (if given)
```

Comment:
I think this doc should say that pathname should always contain the name, because line 21 does the slice(-1) implying that the last item on the path list will always have this special meaning.

🎨 I would also call it pathWithName, since pathname sounds to me as "the name of the path". But this is more like my opinion.

Reply:
I struggled a bit naming this param, because it does _not_ always include the name :/

Imagine there is a job with id `foo.bar.jobname`:
1. when you're on overview path `/`: `pathname` will be `[]`
2. when you're on overview path  `/foo`: `pathname` will be `['foo','bar']`
3. when you're on overview path `/foo.bar`: `pathname` will be `['foo','bar']`
4. when you're on detail page for that job: `pathname` will be `['foo','bar','jobname']`

Now imagine there is a second job with id `nonamespace`, when you're on detail page for _that_ job, `pathname` will be `['nonamespace']` :/

You see, it can contein either only parts of the path, the whole pth, path + name or only the name :confused: